functions/openhabian.bash

```diff
@@ -440,3 +440,32 @@ create_user_and_group() {
     cond_redirect usermod --append --groups openhab,sudo "$userName" &> /dev/null
   fi
 }
+
+## import initial configuration
+##
+##    import_openhab_config()
+##
+## valid argument: file name to import using openhab-cli
+##
+import_openhab_config() {
+  local initialConfig=${initialconfig:-/boot/initial.zip}
+  local restoreFile=${initialConfig}
+
+  echo -n "$(timestamp) [openHABian] Importing initial openHAB configuration... "
+  if [[ $# -eq 1 ]]; then
+    initialConfig=$1
+  fi
```

Comment:
What about https? As a secondary thought, FTP, SFTP, etc.

Reply:
The regexp covers HTTPS, too (and user/pw secured URLs, too). For (S)FTP we would need some other client so I'd rather not like to inflate this.